be/src/agent/heartbeat_server.cpp

```diff
@@ -158,8 +158,10 @@ StatusOr<HeartbeatServer::CmpResult> HeartbeatServer::compare_master_info(const
 
     if (master_info.__isset.backend_ip) {
         if (master_info.backend_ip != BackendOptions::get_localhost()) {
-            LOG(WARNING) << master_info.backend_ip << " not equal to to backend localhost "
-                         << BackendOptions::get_localhost();
+            if (!config::force_use_ip_as_localhost) {
```

Comment:
1. Why is the hostname not accessible, but the IP is accessible? In the K8S env, the IP may be changed sometimes, I think it's not a good choice.
2. We have provided a new internal table named `information_schema.load_tracking_logs` to store the error log.

Reply:
@caneGuy What's your opinion?